src/commands.rs

```diff
@@ -88,29 +88,28 @@ enum RusticCmd {
     /// Check the repository
     Check(CheckCmd),
 
-    /// Copy snapshots to other repositories. Note: The target repositories must be given in the config file!
+    /// Copy snapshots to other repositories
     Copy(CopyCmd),
 
-    /// Compare two snapshots/paths
-    /// Note that the exclude options only apply for comparison with a local path
+    /// Compare two snapshots or paths
     Diff(DiffCmd),
 
-    /// dump the contents of a file in a snapshot to stdout
+    /// Dump the contents of a file within a snapshot to the terminal
```

Comment:
Actually it is stdout and it is a very common use-case to use e.g. `rustic dump | mysql`

Reply:
Yeah, I felt stdout is a relatively technical term, so I thought about making it easier to grasp for newcomers. Wouldn't mind changing it back to `stdout`, it's just less inclusive.

Reply:
I used now both? So it's clear it's on the terminal but also on stdout if someone wants to dig deeper? Is that sufficient?

Reply:
when I think about it, we should rather add a warning (here or in the docu)  that this may produce non-printable characters, so it is advisable to redirect stdout...

Reply:
Ok, reverted to `stdout`